RHEL/6/input/checks/dhcp_server_deny_bootp.xml

```diff
@@ -0,0 +1,31 @@
+<def-group>
+  <definition class="compliance" id="dhcp_server_deny_bootp" version="1">
+    <metadata>
+      <title>Deny BOOTP Requests</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 6</platform>
+      </affected>
+      <description>Prevents the DHCP from responding to 
+      BOOTP requests.</description>
+      <reference source="galford" ref_id="20150317" ref_url="test_attestation" />
+    </metadata>
+    <criteria comment="DHCP is not being used or conditions are met"
+    operator="OR">
+      <extend_definition comment="dhcp package is not installed"
+      definition_ref="package_dhcp_removed" />
+      <criterion comment="Check bootp /etc/dhcp/dhcpd.conf"
+      test_ref="test_dhcp_server_disable_bootp" />
+    </criteria>
+  </definition>
+  <ind:textfilecontent54_test check="all" check_existence="all_exist"
+  comment="Tests that bootp is disabled"
+  id="test_dhcp_server_disable_bootp" version="1">
+    <ind:object object_ref="obj_dhcp_server_disable_bootp" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object id="obj_dhcp_server_disable_bootp"
+  version="1">
+    <ind:filepath>/etc/dhcp/dhcpd.conf</ind:filepath>
+    <ind:pattern operation="pattern match">^[\s]*deny[\s]+bootp\;[\s]*(?:|(?:#.*))?$</ind:pattern>
+    <ind:instance datatype="int">1</ind:instance>
+  </ind:textfilecontent54_object>
+</def-group>
```

Comment:
This OVAL check is good to go... tested with various true&false combinations of ````deny bootp;````